webview/src/experiments/components/table/Cell.tsx

```diff
@@ -31,25 +33,111 @@ const RowExpansionButton: React.FC<RowProp> = ({ row }) =>
     <span className={styles.rowArrowContainer} />
   )
 
-export const FirstCell: React.FC<
-  CellProp & {
-    bulletColor?: string
-    isRowSelected: boolean
-    toggleExperiment: () => void
-    toggleRowSelection: () => void
-    toggleStarred: () => void
+export type RowActionsProps = {
+  isRowSelected: boolean
+  showSubRowStates: boolean
+  starred?: boolean
+  subRowStates: {
+    selections: number
+    stars: number
+    runs: number
+  }
+  toggleRowSelection: () => void
+  toggleStarred: () => void
+}
+
+export const RowAction: React.FC<
+  React.PropsWithChildren & {
+    showSubRowStates: boolean
+    subRowsAffected: number
+    actionAppliedLabel: string
+    hidden?: boolean
   }
 > = ({
-  cell,
-  bulletColor,
+  showSubRowStates,
+  subRowsAffected,
+  actionAppliedLabel,
+  children,
+  hidden
+}) => {
+  return (
+    <div className={cx(styles.rowActions, hidden && styles.hidden)}>
+      <Indicator
+        count={showSubRowStates && subRowsAffected}
+        tooltipContent={
+          showSubRowStates &&
+          subRowsAffected &&
+          `${subRowsAffected} ${pluralize(
+            'sub-row',
+            subRowsAffected
+          )} ${actionAppliedLabel}.`
+        }
+      >
+        {children}
```

Comment:
[I] At the moment only the direct children are being aggregated. Here is a screen recording of an example:

https://user-images.githubusercontent.com/37993418/178470947-92586894-b9e2-45c3-b1cb-2a990f0b7667.mov

When we aggregate from checkpoints to experiments and then up to the current commit we are not taking the cumulative total into account.

Another example:

https://user-images.githubusercontent.com/37993418/178471450-fa337248-913a-43d9-8175-cac4a44be943.mov

I would not even try to include the top level indicator until we have more than one commit/branch in the table. Until that time we will be catering to an edge case.

However, if you feel that they are needed then they need to reflect the totals including all children.

Reply:
I'll add some tests for this current use-case now, but I want to defer this total aggregation for later if it's currently an edge case.